tasks/main.yml

```diff
@@ -1 +1,60 @@
 # role tasks
+
+- name: Check localectl status before
+  command: localectl status
+  register: localectl
+  changed_when: "localectl.rc != 0"
+
+- name: set the localectl before fact
+  set_fact:
+    localectl_before: "{{ localectl.stdout_lines | join ('\n') }}"
+
+- name: Set the language locale
```

Comment:
`set-locale` can set any local var, not just `LANG`, task name should probably just be "Set locale". Bonus points will be awarded for support a list of vars to set, e.g.
```yaml
localectl_locale:
  - LANG=en_US.utf8
  - LC_MESSAGES=en_GB.iso88591
# etc...
```